src/shared/components/notifications/Notifications.tsx

```diff
@@ -26,8 +34,22 @@ const matchGradientToColor = (style: NotificationStyle): Gradients => {
 
 const Notifications: FC = () => {
   const notifications = useSelector(getNotifications)
+  const accountName = useSelector(selectCurrentAccountName)
+  const userJustDeletedAnOrg = Boolean(getFromLocalStorage('justDeletedOrg'))
```

Comment:
Will this run reliably run every time? Is `Notifications.tsx` guaranteed to re-render after the delete operation has been carried out?

Is this a way to get around the lack of any kind of event bus and the delete operation being highly asynchronous?

Reply:
As far as I can tell in remocal testing, this is reliable. I will need to do additional validating in prod behind the feature flag to ensure it functions the same way. 

Notifications.tsx is guaranteed to re-render not due to the delete operation, but due to the quartz redirect that causes an organization switch. That redirect reloads the app, and Notifications along with it. 

To flesh out the thinking here (and provide some alternatives that weren't chosen): 

 Since that switch is a reload, we lose our application state, including redux and context, so dispatching a notification before the reload is a no-go (moreover, closing the overlay triggers a re-render, so the user would see a flash of the notification early, then a reload - have tested this). We're left with either storing the trigger for the reload on the client side but __outside_ of the application state (e.g._ LocalStorage) or server side (send a different value from quartz when a user is redirected after deleting an org). I've opted for the former here since it's simpler than a backend change, and because the notification behaves as expected in my testing (appears once on app reload, then never again until another org is deleted).

If there are lingering concerns: another alternative would be to have the switch occur after a timeout: the notification displays _before_ the org is deleted, then the user gets switched into the default org after a few seconds. Probably more annoying, but more reliable since we're not storing anything in the browser.

Reply:
Cool, thanks for the explanation, and for doing the work required to write the explantion.